website/docs/utilities/ticket-channel.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Ticket Channel
+---
+
+Takes all messages sent in a specified channel, creates a ticket owned by the user who sent it and witht the message as the reason, and deletes the original message.
+
+:::caution
+
+Make sure you have the tickets system set up first.
+
+:::
+
+## Trigger
+
+**Type:** `Regex`<br />
+**Trigger:** `\A`<br />
+**Additional options:** Be sure to restrict it to only run in a specified channel
```

Comment:
Please put more stress on this, as it is more a requirement than an option.
Consider using a warning block.

Reply:
Warning block instead of, or in addition to?